R/map_layout_partition.R

```diff
@@ -84,15 +84,13 @@ layout_tbl_graph_hierarchy <- function(.data, center = NULL,
       Evt2 <- cbind(rep(2, floor(ncol(net)/2)), nrm(rng(floor(mm/2))))
       crd <- rbind(Act, Evt1, Evt2)
       crd[which(is.nan(crd))] <- 0.5
-      crd[, 2] <- crd[, 2] * cos(pi) - crd[, 1] * sin(pi)
       rownames(crd) <- c(dimnames(net)[[1]], dimnames(net)[[2]])
     } else if (center == "events") {
       Act1 <- cbind(rep(0, ceiling(nrow(net)/2)), nrm(rng(ceiling(nn/2))))
       Act2 <- cbind(rep(2, floor(nrow(net)/2)), nrm(rng(floor(nn/2))))
       Evt <- cbind(rep(1, ncol(net)), nrm(rng(mm)))
       crd <- rbind(Act1, Act2, Evt)
       crd[which(is.nan(crd))] <- 0.5
-      crd[, 2] <- crd[, 2] * cos(pi) - crd[, 1] * sin(pi)
```

Comment:
Don't need to make rotated/circular anymore?

Reply:
Not anymore for this layout, I think this was changed in the latest version of `{ggraph}`.